runtime/oti/j9consts.h

```diff
@@ -482,7 +482,7 @@ extern "C" {
 #define J9_GC_POLICY_METRONOME OMR_GC_POLICY_METRONOME
 #define J9_GC_POLICY_NOGC OMR_GC_POLICY_NOGC
 
-// J9_GC_MINIMUM_OBJECT_SIZE does not include dataAddr field
+/* J9_GC_MINIMUM_OBJECT_SIZE does not include dataAddr field */
```

Comment:
Technically this comment itself is not correct. `J9_GC_MINIMUM_OBJECT_SIZE` specifies minimum possible object size and has nothing to do with object headers details. I am suggesting to remove this comment at all

Reply:
And thank you to pay attention to this!

Reply:
I don't see anything fundamentally wrong with the comment itself. From my perspective, it just clarifies that the minimum size doesn't include anything extra, in particular it doesn't (necessarily) include space for the `dataAddr` field.

Reply:
It is wrong fundamentally. It is misleading and unrelated. The definition is not intended to serve any purpose of object internals, this is just global definition used by Mark./Sweep to identify minimum number of object slots (and mark bits) occupied by object.

Reply:
I'm not sure I follow, but I'll just remove the contentious comment.

Reply:
thank you!